src/app/learn/accessibility/code/index.html

```diff
@@ -0,0 +1,39 @@
+<stache
+  pageTitle="Coding for accessibility"
+  showTableOfContents="false"
+  navTitle="Coding"
+  navOrder="2">
+  
+  <stache-page-summary>
+    <p>The following best practices help to ensure that SKY UX supports accessibility features and meets WCAG 2.0 Level A and AA success criteria.</p>
+  </stache-page-summary>
+  
+  <ul>
+    <li>
+      <p>Use structural semantic markup. This ensures that the role of elements and their relationships is available programmatically to user agents that help to communicate the content's meaning.</p>
+      <ul>
+        <li>
+          <p><a href="http://learn.shayhowe.com/html-css/getting-to-know-html/">HTML semantics overview</a></p>
+        </li>
+        <li>
+          <p><a href="http://learn.shayhowe.com/advanced-html-css/semantics-accessibility/">Text-level semantics, microdata, and WAI-ARIA roles</a></p>
+        </li>
+      </ul>
+    </li>
+    <li>
+      <p>Apply <a href="http://webaim.org/articles/">accessible techniques to HTML</a> forms, tables, links, media, and scripting, as well as to keyboard use.</p>
```

Comment:
@Blackbaud-MattGregg, I pointed out in the old PR for the skyux2 repo that this link takes users to a list of links and asked if there is a particular article that we should point to. You replied that you updated with links to specific articles, but that change never showed up in the old PR. Can you pass along whatever change you meant to make so that I can make it here in this new PR?